src/Dynatrace.OpenTelemetry.Exporter.Metrics.Tests/DynatraceMetricsExporterTests.cs

```diff
@@ -663,35 +636,42 @@ public async Task Export_ObservableDoubleCounter()
 
 			var sut = new DynatraceMetricsExporter(null, null, new HttpClient(mockMessageHandler.Object));
 
-			// Delta is already the preferred temporality for our exporter
+			// Delta is already the preferred/temporality temporality for our exporter
 			var metricReader = new TestMetricReader(sut);
 			using var provider = Sdk.CreateMeterProviderBuilder()
 				.AddMeter(meter.Name)
 				.AddReader(metricReader)
 				.Build();
 
+			var i = 1;
 			var observableCounter = meter.CreateObservableCounter("double_obs_counter", () =>
 			{
 				return new List<Measurement<double>>()
 				{
-					new Measurement<double>(10.3, _attributes)
+					new(i++ * 10.3, _attributes)
```

Comment:
does this create two entries? not sure what is expected here

Reply:
This just increments `i` then multiple it to 10, in order to see if deltas are being calculated correctly.

- The first time `i=1` so it will produce a value of `10.3` and increment `i=2`
- The second time `i=2` so it will produce a value of `20.3` 

Then I "export" twice (by calling the collect on the reader)

```csharp
metricReader.Collect();
await AssertLines(metricReader, 10.3);
```
So the next time I export, it will produce `20.3` but because the SDK calculates the delta, I expect to receive `10.3` again.

Reply:
I will add some comments to be make it more obvious what's going on:

```charp
// When exported twice, this obs counter will output
// 10.3 -> 20.6. Since we expect deltas to be calculated by the SDK
// we assert both exports to be 10.3
```

Reply:
Thats what i thought, thanks for clarifying it!